porch/pkg/registry/porch/functions.go

```diff
@@ -48,6 +48,8 @@ func (f *functions) New() runtime.Object {
 	return &v1alpha1.Function{}
 }
 
+func (f *functions) Destroy() {}
```

Comment:
Why is this Destroy method now required with a cli-utils update?

Is it ok for it to not be implimented?

/cc @justinsb

Reply:
Can you add comments to the empty methods to explain that? I'm surprised the linter isn't complaining about that, since they're public methods.